docs/docsite/rst/community/collection_contributors/collection_requirements.rst

```diff
@@ -37,17 +36,17 @@ As with any project it is very important that we get feedback from users, contri
 Keeping informed
 ================
 
-You should subscribe to:
+To track changes that affect collections:
 
-* The `news-for-maintainers repository <https://github.com/ansible-collections/news-for-maintainers>`_ to track changes that collection maintainers should be aware of. Subscribe only to issues if you want less traffic.
-* The `Bullhorn <https://forum.ansible.com/c/news/bullhorn/17>`_ Ansible contributor newsletter.
+* Join the `Collection Maintainers & Contributors forum group <https://forum.ansible.com/g/CollectionMaintainer>`_.
+* Subscribe to the `news-for-maintainers repository <https://github.com/ansible-collections/news-for-maintainers>`_. Subscribe only to issues if you want less traffic.
```

Comment:
We should really replace this repo with the corresponding tag in the Forum. New announcements are no longer posted in the repo (except by the Core team, and it seems they are stopping that in general).

Reply:
it's still being discussed about the repo but i've removed the line if the announcements are forwarded to the forum anyway, PTAL